fabric-model-events-api-v1/src/client/resources/fabric-model-events-api-v1.accesswidener

```diff
@@ -0,0 +1,4 @@
+accessWidener v2 named
```

Comment:
This AW does not seem to be declared in FMJ. Also, the build.gradle path is wrong.